pvnet/app.py

```diff
@@ -195,21 +196,37 @@ def app(
 
     # ---------------------------------------------------------------------------
     # 1. Prepare data sources
-    logger.info("Loading GSP metadata")
 
-    ds_gsp = next(iter(OpenGSPFromDatabase()))
+    # Make pands Series of most recent GSP effective capacities
+    logger.info("Requesting GSP effective capacities from PVLive")
 
-    # DataFrame of most recent GSP capacities
-    gsp_capacities = (
-        ds_gsp.sel(
-            time_utc=t0,
-            method="ffill",
-        )
-        .sel(gsp_id=gsp_ids)
-        .to_dataframe()
-        .capacity_megawatt_power
```

Comment:
Could you add a comment here that when we have `capacity_mwp` in our database, we should use that